implementations/rust/daemon/src/node.rs

```diff
@@ -7,17 +7,24 @@ use crate::cli;
 use crate::config::{Config, Role};
 use crate::worker::Worker;
 
+use crate::cli::VaultKind;
 use ockam_channel::*;
-use ockam_kex::{
-    xx::{XXInitiator, XXNewKeyExchanger, XXResponder},
-    CipherSuite,
-};
+use ockam_kex::xx::{XXInitiator, XXNewKeyExchanger, XXResponder};
 use ockam_message::message::AddressType;
 use ockam_router::router::Router;
 use ockam_system::commands::{OckamCommand, RouterCommand};
 use ockam_transport::transport::UdpTransport;
+use ockam_vault::software::DefaultVaultSecret;
 use ockam_vault::types::*;
-use ockam_vault::{file::FilesystemVault, DynVault};
+use ockam_vault::{file::FilesystemVault, DynVault, Secret};
+use cfg_if::cfg_if;
+
+cfg_if! {
```

Comment:
```suggestion
#[cfg(feature = "atecc608a)]
use ockam_vault::c::Atecc608aVaultBuilder;
#[cfg(feature = "atecc608a)]
use ockam_vault::c::CVault;
```

Removes the dependency on cfg_if

Reply:
Why don't we want to depend on cfg_if?